raw_data/preprocessed/africa/seychelles_english.txt

```diff
@@ -0,0 +1,91 @@
+
+(8) [1] The Minister, on the recommendations of the Authority may, declare by notice in the gazette any area as an air pollution control area for the purposes of this Part. [2] No person shall, without an authorisation from the Authority, cause any emission in an air pollution control area. [3] No person shall, without an authorisation from the Authority, establish or operate any industrial plant in an air pollution control area. [4] The Authority may grant the authorisation for such period and with such conditions as it may specify. [5] An authorisation may be revoked at any time if the conditions under which it was granted are violated. [6] The Minister may by regulations: [a] prescribe controls for the use of any fuel or any appliance which may cause or is likely to cause air pollution; [b] prohibit the burning of any material (not being fuel), in an air pollution control area. [7] For the purpose of this Part the Authority may establish monitoring stations or networks to locate sources of atmospheric pollution. [8] A person who contravenes subsection [2] or [3] is guilty of an offence under this Act.
+
+(9) [1] The Minister may, on the recommendations of the Authority, by regulations, establish noise emission standards and such noise abatement measures as may be necessary to preserve the environment. [2] No person shall emit noise in excess of the noise emission standards established under this Part, except where an authorisation has been granted by the Authority allowing excessive emission of noise under such terms and conditions as it may determine. [3] The terms and conditions of an authorisation under sub-section [2] shall include compliance with such measures required to adequately protect persons exposed to excessive levels of noise. [4] A person who contravenes subsection [2] is guilty of an offence under this Act.
```

Comment:
Please replace `sub-section` with `subsection` (the latter is used 34 times in this legislation, while the former is only used four times).